### PR TITLE
Formula update for conductr-cli 1.2.0

### PR DESCRIPTION
**Pull request by fsat**

### CODE REVIEW

**Comment by fsat** [comment]
Tested the URL displayed on https://github.com/typesafehub/homebrew-conductr/pull/3/files#diff-65f7cec3251fc063df0b0b284992bd81R4 - the correct zip file is downloaded.

File: Formula/conductr-cli.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class ConductrCli < Formula
   desc "The Lightbend ConductR CLI"
   homepage "https://conductr.lightbend.com"
-  url "https://bintray.com/lightbend/generic/download_file?file_path=conductr-cli-1.1.0-Mac_OS_X-x86_64.zip"
-  version "1.1.0"
-  sha256 "9b169b24be1173dde257bdd633df8df7066fe015e3eefd8ed4d36f41f3e59852"
+  url "https://bintray.com/lightbend/generic/download_file?file_path=conductr-cli-1.2.0-Mac_OS_X-x86_64.zip"
+  version "1.2.0"
+  sha256 ""
```
Comment (TimMoore) [inline Formula/conductr-cli.rb:6]: No hash?

Comment (huntc) [reply Formula/conductr-cli.rb:6]: Whoops!

**Comment by TimMoore** [comment]
```
$ brew upgrade
==> Upgrading 1 outdated package, with result:
typesafehub/conductr/conductr-cli 1.2.0
==> Upgrading typesafehub/conductr/conductr-cli 
==> Downloading https://bintray.com/lightbend/generic/download_file?file_path=conductr-cli-1.2.0-Mac_OS_X-x86_64.zip
==> Downloading from https://dl.bintray.com/lightbend/generic/conductr-cli-1.2.0-Mac_OS_X-x86_64.zip
######################################################################## 100.0%
Warning: Cannot verify integrity of conductr-cli-1.2.0.zip
A checksum was not provided for this resource
For your reference the SHA256 is: 8e2b4c62085684813e3b75f7dfb8ead4a13f21a9968364ee88ce7c61b7921d3e
🍺  /usr/local/Cellar/conductr-cli/1.2.0: 6 files, 22.7MB, built in 27 seconds
```

File: Formula/conductr-cli.rb (hunk 1 above)
Comment (fsat) [reply Formula/conductr-cli.rb:6]: Ok, not sure why there's no hash - I need to investigate.

**Pull request merged**